modal/_output.py

```diff
@@ -81,22 +82,27 @@ def download_progress_bar() -> Progress:
     )
 
 
-class LineBufferedOutput(io.StringIO):
+class LineBufferedOutput:
```

Comment:
It wasn't quite a `StringIO` even before, and definitely isn't one with this change of `write`, and I don't think we need it to be?

Reply:
Yeah I think it makes sense to remove the base class